plugins/service/renderer/srv6/srv6_renderer_test.go

```diff
@@ -217,6 +282,32 @@ func TestServiceWithRemoteBackend(t *testing.T) {
 	closeResources(data)
 }
 
+func TestServiceWithRemoteBackendIPv4(t *testing.T) {
+	RegisterTestingT(t)
+	retriever := configRetriever.NewMockConfigRetriever()
+	data := initTest("TestServiceWithRemoteBackend", defaultConfig(true), retriever, false)
+
+	// setup service
+	emptyResync(data)
+	service1 := addServiceMetadata(data, defaultPort)
+	pod2IP := addLocalPod(renderer_testing.Pod2, data, retriever)
```

Comment:
In the light of my previous comment about pod IP creation, my first thought is to rename this too, but idea is to test remove backend and not local backend. So pod2 shouldn't be in test at all. This test is copy of TestServiceWithRemoteBackend where it is done this way (and i did that test that nonclean way). Now is probably that time to refactor and simplify both test by removing local backend. The main work will be in refactoring addServiceEndpointsInMultipleNodes so it supports also one endpoint and not fixed 2 endpoints.

Reply:
Done